Babylon.js compatibily lists (#559)
BabylonJS/BabylonReactNative: README.md

For iOS the XCode project needs to be generated with `CMake` as described [above](#ios) and added to your `xcworkspace`.

> ## Supported React-Native

Reviewer: ```suggestion
## Supported Versions
```

> The React-native 
> In-between versions not in this list (like 0.66, 0.67, ...) use the previous React-Native version (0.65 in that case)

Reviewer: ```suggestion
When using a React Native version in between the versions listed, use the previous React Native version. For example, React Native version 0.66 or 0.67 should use Babylon React Native versions that correspond to 0.65.
```

> | React-Native version | BabylonReactNative versions |

Reviewer: ```suggestion
| React Native | Babylon React Native |
```

|---|---|
| 0.64 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
| 0.65 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
| 0.69 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
| 0.70 | 1.5.0, 1.5.1 |

## Supported Babylon.js versions

> Depending on the BabylonReactNative NPM package version, some Babylon.js NPM dependency version may or may not be compatible. Here is a compatibility list that has been tested:

Reviewer: ```suggestion
Depending on the Babylon React Native NPM package version, some Babylon.js NPM dependency versions may or may not be compatible. Here is a compatibility list that has been tested:
```

> | BabylonReactNative version | Babylon.js versions |

Reviewer: ```suggestion
| Babylon React Native | Babylon.js |
```

For iOS the XCode project needs to be generated with `CMake` as described [above](#ios) and added to your `xcworkspace`.

## Supported Versions

When using a React Native version in between the versions listed, use the previous React Native version. For example, React Native version 0.66 or 0.67 should use Babylon React Native versions that correspond to 0.65.

| React Native | Babylon React Native |
|---|---|
| 0.64 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
| 0.65 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
| 0.69 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
| 0.70 | 1.5.0, 1.5.1 |

> 3 NPM Packages are available: 1 mandatory for Windows or Android/iOS, 1 for Windows and 1 for Android/iOS.

Reviewer 2: Suggestion (clarity):
`1 mandatory for Windows or Android/iOS` -> `1 mandatory for all platforms`

Babylon React Native NPM Package naming convention is 
- `@babylonjs/react-native` for the base package
- `@babylonjs/react-native-windows-0-XX` for Windows

> - `@babylonjs/react-native-iosandroid-0-XX` for iOS/Android

Reviewer: FYI, we also have some docs in the package readme: https://github.com/BabylonJS/BabylonReactNative/tree/master/Modules/%40babylonjs/react-native#platform-native-packages

XX corresponds to the React Native version. `@babylonjs/react-native-windows-0-64` corresponds to React Native 0.64.

> ## Supported Babylon.js versions

Reviewer: nit:
```suggestion
## Supported Babylon.js Versions
```

Depending on the Babylon React Native NPM package version, some Babylon.js NPM dependency versions may or may not be compatible. Here is a compatibility list that has been tested:

> | Babylon React Native | Babylon.js |

Reviewer 2: Suggestion (clarity):
`| Babylon.js |` -> ``| Babylon.js `(@babylonjs/core)` |``

For iOS the XCode project needs to be generated with `CMake` as described [above](#ios) and added to your `xcworkspace`.

## Supported Versions

> 
> When using a React Native version in between the versions listed, use the previous React Native version. For example, React Native version 0.66 or 0.67 should use Babylon React Native versions that correspond to 0.65.

Reviewer: This statement isn't necessary anymore right?
```suggestion
```
